internal/pkg/service/appsproxy/dataapps/api/api.go

```diff
@@ -0,0 +1,4 @@
+// Package api provides an interface for data apps API.
+package api
+
```

Comment:
Create `api` pkg to split: 
1. API calls layer.
2. And caching/throttling layer.